app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -303,11 +304,12 @@ class SuggestedEditsTasksFragment : Fragment() {
     }
 
     private fun showOneTimeSequentialUserStatsTooltips() {
-        val balloon = FeedbackUtil.showTooltip(requireContext(), contributionsStatsView.tooltipText)
+        val arrowOrientation = ArrowOrientation.BOTTOM
```

Comment:
If we have already imported it, is it necessary to create an extra `val` for it instead of using it directly?